Nodes/config/routes.php

```diff
@@ -14,17 +15,6 @@
     $routeBuilder->connect('/promoted/*', ['controller' => 'Nodes', 'action' => 'promoted']);
     $routeBuilder->connect('/search/*', ['controller' => 'Nodes', 'action' => 'search']);
 
-    $routeBuilder->scope('/:type', function (RouteBuilder $routeBuilder) {
-        $routeBuilder->routeClass('Croogo/Taxonomy.TypeRoute');
-        $routeBuilder->connect('/', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-        $routeBuilder->connect('/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
-    });
-
     // Content types
-    $routeBuilder->connect('/node', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-    $routeBuilder->connect('/node/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
+    CroogoRouter::routableContentTypes($routeBuilder);
```

Comment:
In theory it should work :) There was an issue with caches not being cleared (Due to the cache behavior of 2.x not having been migrated). But this has been fixed, just note that there is a bug in CakePHP that will cause the cache clearing to fail if you've got debugKit enabled (PR to fix it is https://github.com/cakephp/cakephp/pull/9035)